.github/workflows/hivemq-operator-integration-test.yml

```diff
@@ -50,20 +50,19 @@ jobs:
         with:
           java-version: '11'
           distribution: 'temurin'
-      - name: Run HiveMQ Operator (legacy) integration tests
-        uses: gradle/gradle-build-action@e2097ccd7e8ed48671dc068ac4efa86d25745b39 # v3
-        env:
-          TEST_PLAN: ${{ matrix.test-plan }}
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@db19848a5fa7950289d3668fb053140cf3028d43 # v3
         with:
-          cache-disabled: true
```

Comment:
Not sure about this one. By default the cache is enabled using this GitHub actions plugin and to be honest I'm not completely sure why it was disabled in the first place.
If I'm not wrong this is just to cache some of the Gradle tasks, so this shouldn't have any impact on the integration tests other than speeding up the build, right?

Reply:
Yeah, this should just speed up a consecutive execution by skipping Gradle tasks that have unchanged inputs. Since runners are always freshly started, I see no way how this should create an unwanted cache usage between jobs.